### ISSUE TITLE
Test runs are slow

### ISSUE THREAD

**Issue by earnubs** [canonical-ols/javan-rhino#37]
```
time npm test
[...]
real    0m23.280s
user    0m22.780s
sys     0m1.958s
```

**Comment by bartaz** [comment]
I'm not sure but it may be due to the way files are now compiled for tests by babel with webpack config. Because it seems it 'hangs' for a while and then actual tests run quickly (especially that we don't test much yet).

**Comment by bartaz** [comment]
I verified it locally.

With current setup (`babel-plugin-webpack-loaders` plugin to load CSS modules in tests):

```
npm test  27.00s user 2.54s system 102% cpu 28.693 total
```

With disabled `babel-plugin-webpack-loaders` plugin (and CSS modules mocked to empty objects in JS files):

```
npm test  5.61s user 0.44s system 107% cpu 5.645 total
```

**Comment by earnubs** [comment]
fixed in #49
